share/completions/pg_dump.fish

```diff
@@ -0,0 +1,60 @@
+complete -c pg_dump --no-files -a '(__fish_complete_pg_database)'
+
+# General options:
+complete -c pg_dump -s f -l file -r -d "output file or directory name"
+complete -c pg_dump -s F -l format -x -a "c d t p" -d "output file format"
+complete -c pg_dump -s j -l jobs -x -d "use this many parallel jobs to dump"
+complete -c pg_dump -s v -l verbose -d "verbose mode"
+complete -c pg_dump -s V -l version -d "output version information, then exit"
+complete -c pg_dump -s Z -l compress -x -d "compression level for compressed formats"
+complete -c pg_dump -l lock-wait-timeout -x -d "fail after waiting TIMEOUT for a table lock"
+complete -c pg_dump -l no-sync -d "do not wait for changes to be written safely to disk"
+complete -c pg_dump -s '?' -l help -d "show this help, then exit"
+
+# Options controlling the output content:
+complete -c pg_dump -s a -l data-only -d "dump only the data, not the schema"
+complete -c pg_dump -s b -l blobs -d "include large objects in dump"
+complete -c pg_dump -s B -l no-blobs -d "exclude large objects in dump"
+complete -c pg_dump -s c -l clean -d "clean (drop) database objects before recreating"
+complete -c pg_dump -s C -l create -d "include commands to create database in dump"
+complete -c pg_dump -s E -l encoding -x -d "dump the data in encoding ENCODING"
+complete -c pg_dump -s n -l schema -x -d "dump the named schema(s) only"
```

Comment:
Just use the plural - "Dump the named schemas only".

People will get that they can also use one, and it's a bit shorter and easier to skim.